tests/test_config.py

```diff
@@ -134,3 +134,9 @@ def test_sources_invalid(self):
         self.assert_spellcheck(
             '.source.yml', self.bad_words2, names=['name', 'other_name'], sources=[self.tempdir + '/**/test1.txt']
         )
+
+    def test_no_matrix(self):
+        """Test a Value error is raised if the configuration has no matrix."""
```

Comment:
Let's change `Value error` to `` `ValueError` `` in the docstrings. The back ticks will allow the term to be ignored in the docstrings spell check.

Reply:
thx will do

Reply:
done